python/strings_udf/cpp/include/cudf/strings/udf/char_types.cuh

```diff
@@ -183,6 +183,30 @@ __device__ inline bool is_lower(cudf::strings::detail::character_flags_table_typ
     flags_table, d_str, string_character_types::LOWER, string_character_types::CASE_TYPES);
 }
 
+/**
+ * @brief Returns true if string is in title case
+ *
+ * @param tables The char tables required for checking characters
+ * @param d_str Input string to check
+ * @return True if string is in title case
+ */
+__device__ inline bool is_title(cudf::strings::detail::character_flags_table_type* flags_table,
+                                string_view d_str)
+{
+  bool valid                 = false;  // requires one or more cased characters
```

Comment:
Can we add a test case that covers the branch where we start with valid and never enter the `if` statement below? I assume a numeric like `"123"` or a symbolic string like `"^#("` would do the trick.

Reply:
Not sure whether it's best to test in the string UDF tests, the cudf tests, or both; up to you.